packages/devtools_app/lib/src/debugger/file_search.dart

```diff
@@ -0,0 +1,146 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
```

Comment:
I just copied this from another file. How do the licenses work? Should this be changed to 2021?

Reply:
For new files, copy the license and change the year to current. For existing files, we leave the year as is.

Reply:
Done, thanks!